examples/full-example/.mendelrc

```diff
@@ -53,23 +53,19 @@ transforms: # a list of all available transforms for all envs and types
         - istanbul
 
 types:
-  json:
+  json: # Type id. Doesn't have to be file extension.
```

Comment:
Same suggestion as `transforms`. One or two lines before "types:" explaining why types exist, and document only first one with column aligned comments.